tools/payloads/configbytes/apk_configbytes_extract.rb

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env ruby
+# -*- coding: binary -*-
+
+class ExtractError < StandardError; end
+class HelpError < StandardError; end
+class UsageError < ExtractError; end
+
+require 'rex'
+require 'tmpdir'
+require 'optparse'
+require 'open3'
+require 'nokogiri'
+require 'fileutils'
+
+def run_cmd(bin, args, verbose)
```

Comment:
Args should be an array here and it should not be joined to cmd as a string. The `Open3` methods allow specifying the argv value as a native ruby array. This has two benefits, one is that you don't need to worry about strings with spaces in them and the other is that it's not able to have arguments injected into it which could potentially cause security issues.